toolchain/check/testdata/let/fail_modifiers.carbon

```diff
@@ -0,0 +1,95 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+
+// CHECK:STDERR: fail_modifiers.carbon:[[@LINE+3]]:1: ERROR: Semantics TODO: `private`.
+// CHECK:STDERR: private let a: i32 = 1;
+// CHECK:STDERR: ^~~~~~~
+private let a: i32 = 1;
```

Comment:
Should this implementation TODO be separated from the structural errors?

(also applies to other similar tests)